### DISCUSSION TITLE
MOOSE Docs websites rendering drop-down menus on mobile

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#23928] [Q&A Tools] [answered]
Hello,

Today, I made live Ferret's website https://mangerij.github.io/ferret/index.html and noticed that the drop-down menus do not render correctly (and are not able to be clicked) if one uses a mobile browser. I checked and this seems to be a MOOSE Docs wide issue since both the drop-down menu on the framework's website as well as other MOOSE-based apps (for example Raccoon, https://hugary1995.github.io/raccoon ) also have this issue. It does not seem to be an issue with the browser used on mobile. I checked Chrome, Firefox, and others and all of the drop-down menus cannot be expanded and clicked on. I figure I would just raise this issue - but it would be nice if there was a work-around...

**Comment by GiudGiud** [comment, marked as answer]
Can confirm, creating an issue
